src/Modules/CalcOffence.lua

```diff
@@ -2740,6 +2748,15 @@ function calcs.offence(env, actor, activeSkill)
 				t_insert(breakdown.AverageDamage, s_format("= %.1f", output.AverageDamage))
 			end
 		end
+		if globalBreakdown then
+			if globalOutput.AverageBurstDamage > 0 then
```

Comment:
```suggestion
		if globalBreakdown and globalOutput.AverageBurstDamage > 0 then
```
No need for two if statements.